runtime/gpufortrt/src/gpufortrt_api.cpp

```diff
@@ -37,11 +37,11 @@ size_t gpufortrt_get_property(int dev_num,
       HIP_CHECK(hipMemGetInfo(&free, &total))
       return total;
       break;
-    case gpufortrt_free_memory:
-      size_t free;
-      size_t total;
-      HIP_CHECK(hipMemGetInfo(&free, &total))
-      return free;
+    case gpufortrt_property_free_memory:
+      size_t freeMem;
```

Comment:
Why the renaming? Usage of camelCase does not fit with rest of code base. I am planning to adopt the LLVM programming style eventually but not for the time being.

Reply:
In switch case, the scope of variables is the switch not the cases: we get errors about the redefinition of variables free and total.

Either we have to put the definition of free and total in the switch (which looks weird) or we need to define new variables. How about free_mem and total_mem?

Reply:
fixed in commit https://github.com/ROCmSoftwarePlatform/gpufort/pull/45/commits/59d4293c415a47f93867135dddb827fb1fd7a4ae

Reply:
Two more things:
1. Probably makes sense in this case to declare the inout arguments outside
of the scope to prevent naming conflicts.
2. Alternatively, you can use '{' and '}' like this
`case <label> { ... }`  ( don't forget to break )
to have a local scope for variables within the different cases where needed.

Reply:
Took first option and fixed in commit https://github.com/ROCmSoftwarePlatform/gpufort/pull/45/commits/a69f450319a5cea333802d132366ac30cccc298a